<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Icex-set</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Файл для фона рабочего стола</string>
    </property>
    <widget class="QLineEdit" name="DesktopImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>371</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_DesktopImage">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Обзор</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_SaveApp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить и применить</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DesktopCentr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Отображать файл фона рабочего стола по центру</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_DesktopScaled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Файл фона рабочего стола масштабируется на весь экран</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
